sources/libertinussans-bold.sfd

```diff
@@ -22906,39 +22906,11 @@ EndChar
 
 StartChar: uni2127
 Encoding: 8487 8487 1106
-Width: 666
+Width: 766
 Flags: MW
 LayerCount: 2
 Fore
-SplineSet
-549 651 m 2
- 570 651 593 652 609 654 c 1
- 611 651 l 1
- 609 623 609 569 611 554 c 1
- 609 550 l 1
- 609 550 551 572 417 572 c 1
- 423 514 l 1
- 550 489 630 402 630 274 c 0
- 630 111 494 -7 297 -7 c 0
- 98 -7 -36 91 -36 274 c 0
- -36 394 40 491 171 514 c 1
- 174 572 l 1
- 40 572 -18 550 -18 550 c 1
- -20 554 l 1
- -18 569 -18 623 -20 651 c 1
- -18 654 l 1
- -2 652 21 651 42 651 c 2
- 210 651 l 2
- 222 651 232 646 232 637 c 2
- 224 488 l 1
- 142 447 111 382 111 274 c 0
- 111 133 167 42 297 42 c 0
- 436 42 483 145 483 273 c 0
- 483 385 462 426 370 488 c 1
- 362 636 l 2
- 362 645 373 651 384 651 c 2
- 549 651 l 2
-EndSplineSet
+Refer: 704 937 S -1 0 0 -1 765.999 651 2
```

Comment:
Hmm, the references are selected but `make check` didn’t catch that!

Reply:
Bug in the normalizer, please rebase on master and normalize again.

Reply:
Ok, master updated, I'll rebase and re-normalise.